sonar-dotnet-shared-library/pom.xml

```diff
@@ -14,7 +14,7 @@
   <description>Share code between C# and VB.NET plugins</description>
 
   <properties>
-    <sonar.version>7.4-alpha1</sonar.version>
+    <sonar.version>7.6</sonar.version>
```

Comment:
I have updated the API just to have the latest Dependency Injection related code (see the commit "Read the analyzeGenerateCode setting in the plugin").

One could argue I could drop this change from this PR and make it separately.